docs/install-upgrade/build-wiring.md

```diff
@@ -56,28 +54,34 @@ OPTIONS:
 ```
 
 ### Sample Switch Configuration
-```yaml
+``` { .yaml .annotate linenums="1" }
 apiVersion: wiring.githedgehog.com/v1beta1
 kind: Switch
 metadata:
   name: ds3000-02
 spec:
   boot:
-    serial: ABC123XYZ
+    serial: ABC123XYZ # (1)
   role: server-leaf
   description: leaf-2
-  profile: celestica-ds3000
-  portBreakouts:
+  profile: celestica-ds3000 # (2)
+  portBreakouts: # (3)
     E1/1: 4x10G
     E1/2: 4x10G
     E1/17: 4x25G
     E1/18: 4x25G
     E1/32: 4x25G
-  redundancy:
+  redundancy: # (4)
     group: mclag-1
     type: mclag
 ```
 
+1. Can also use `mac: 34:AD:61:00:02:03` of 1GbE interface
+2. See the [list](../reference/profiles.md) of profile names
```

Comment:
I think something like links to the profiles is a perfect fit for the anotations